app/src/main/java/com/cornellappdev/uplift/data/adapters/ApolloAdapters.kt

```diff
@@ -241,29 +241,30 @@ fun GymListQuery.GetAllGym.pullEquipmentGroupings(
 
     val equipments = facilityIn.facilityFields.equipment ?: return listOf()
 
-    val equipmentGroups = HashMap<EquipmentType, EquipmentGrouping>()
-    equipments.forEach { equipment ->
-        if (equipment != null) {
-            val equipType = equipment.equipmentFields.equipmentType
-            val equipmentF = EquipmentField(
-                id = equipment.equipmentFields.id,
-                accessibility = equipment.equipmentFields.accessibility,
-                name = equipment.equipmentFields.name,
-                facilityId = equipment.equipmentFields.facilityId,
-                quantity = equipment.equipmentFields.quantity ?: 0,
-            )
-
-            if (equipmentGroups.containsKey(equipType)) {
-                equipmentGroups[equipType]?.equipmentList?.add(equipmentF)
-            } else {
-                equipmentGroups[equipType] = EquipmentGrouping(
-                    equipmentType = equipType,
-                    equipmentList = ArrayList<EquipmentField>()
-                )
-                equipmentGroups[equipType]?.equipmentList?.add(equipmentF)
-            }
-        }
-    }
+    val equipmentGroups = HashMap<MuscleGroup, EquipmentGrouping>()
+    /* TODO: Update Equipment Logic to match new schema */
```

Comment:
Oh that's a shame. Backend is changing the equipment schema? I guess then this TODO is the best we can do for now.